[ESS][8.13] Indicator match rule alert suppression docs (#4888)
elastic/security-docs: docs/detections/alert-suppression.asciidoc

. When configuring the rule type (the *Define rule* step for a new rule, or the *Definition* tab for an existing rule), specify how you want to group events for alert suppression:
+
--
* Custom query rule: In *Suppress alerts by*, enter 1-3 field names to group events by the fields' values. 
* Threshold rule: In *Group by*, enter up to 3 field names to group events by the fields' values, or leave the setting empty to group all qualifying events together. 
--
+
NOTE: If you specify a field with multiple values, an alert grouping is created for each value. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts will be suppressed separately for each value of `127.0.0.1`, `127.0.0.2`, and `127.0.0.3`. 

* Indicator match rule: In *Suppress alerts by*, enter 1-3 field names to group events by the fields' values. 
+
NOTE: If you specify a field with multiple values, an alert grouping is created for alerts that contain the array you specified. For example, if you suppress alerts by `destination.ip` of `[127.0.0.1, 127.0.0.2, 127.0.0.3]`, alerts that contain this array are grouped and suppressed.

> 

Reviewer: Wondering whether we can improve this section a bit

<img width="839" alt="Screenshot 2024-03-11 at 10 17 29" src="https://github.com/elastic/security-docs/assets/92328789/215dca4f-e77b-482c-af6e-166f252007f6">

First not is applicable for both query and threshold rule types.

Second note - only to IM rule type

There is an Intendation  for a second note, but would it be understood by users the first not applied to query rule as well?

Author: Combined the two notes and updated the format in [943cc12](https://github.com/elastic/security-docs/pull/4888/commits/943cc12315b52e4523e741e1182a476ad84a288c). 

@vitaliidm let me know if the changes work - thanks!

image::images/timeline-button.png[Investigate in timeline button, 200]

* Alert details flyout — Select *Take action* -> *Investigate in timeline*.

=== Alert suppression limit by rule type

> Except for custom query rules (which don't have an alert suppression limit), each rule type has a maximum number of alerts that can be suppressed:

Reviewer 2: Maybe reorder; the focus point (each rule type has a maximum) feels a little buried behind two different qualifying phrases.
```suggestion
Some rule types have a maximum number of alerts that can be suppressed (custom query rules don't have a suppression limit):
```

> * **Threshold** - The maximum number of alerts that can be suppressed is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100`.

Author: ```suggestion
* **Threshold** - The maximum number of is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.
```

> * **Indicator match** - The maximum number is five times the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `500`.

Author: ```suggestion
* **Indicator match** - The maximum number is five times the value specified for the <<opt-fields-all,`max_signals`>> setting. The default `max_signals` value is `100`, which means the default maximum limit for indicator match rules is `500`.
```

image::images/timeline-button.png[Investigate in timeline button, 200]

* Alert details flyout — Select *Take action* -> *Investigate in timeline*.

=== Alert suppression limit by rule type

Some rule types have a maximum number of alerts that can be suppressed (custom query rules don't have a suppression limit):

> * **Threshold** - The maximum number of is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.

Reviewer 3: ```suggestion
* **Threshold** - The maximum number of alerts is the value specified for the <<opt-fields-all,`max_signals`>> setting, which is `100` by default.
```
